### PR TITLE
CDI homepage and styling

### PR DESCRIPTION
**Pull request by stephen-mcneal**

### CODE REVIEW

File: dashboard/static/css/style.css [hunk 1]
```diff
@@ -15,19 +15,73 @@ body {
...
+	padding:20px
+
+}
+.section4 {
+	width:650px;
+	height:500px;
+	
+	padding:20px
+
+}
```
Comment (xhagrg) [inline dashboard/static/css/style.css]: These can be consolidated into one:
```
.section1, .section2, .section3, .section4 {
    ...
}
```

File: dashboard/static/css/style.css [hunk 2]
```diff
@@ -181,4 +235,113 @@ body {
...
+.section4 {
+	width:60%;
+	height:400px;
+	
+	padding:20px
+
+}
```
Comment (xhagrg) [inline dashboard/static/css/style.css:260]: Why update the same class?

File: dashboard/templates/HOMEPAGE.html [hunk 3]
```diff
@@ -2,6 +2,109 @@
...
+<div class="section2">
+	<div class="information">
+		<br>
+		<br>
+		<br>
+		<br>
+		<br>
+		<h1 style="width:75%;text-align:right"># in Climate Collection | {{ climate_collection_count }}</h1>
+		
+		<h1 style="width:75%;text-align:right" ># in Masterlist | {{ masterlist_count }}</h1>
```
Comment (xhagrg) [inline dashboard/templates/HOMEPAGE.html]: Needs indentation.

File: dashboard/views.py [hunk 4]
```diff
@@ -1,7 +1,10 @@
 from django.shortcuts import render
 from django.views.generic import TemplateView
 from django.views import View
-
+import pandas as pd
+import json
+import plotly
+import plotly.express as px
```
Comment (xhagrg) [inline dashboard/views.py]: move imports before from and alphabetize them.

File: dashboard/views.py [hunk 5]
```diff
@@ -15,13 +18,28 @@ def get(self, request):
...
+            dct = {k:[v] for k,v in item.items()}
+            timeseries_df=timeseries_df.append(pd.DataFrame.from_dict(dct))
+
+        timeseries_df.columns=['Date', "Masterlist", "Climate Collection"]
+        fig = px.line(timeseries_df, x='Date', y=timeseries_df.columns,
+              hover_data={"Date": "|%B %d, %Y"},
+              title='Timeseries')
+        graphJSON = json.dumps(fig, cls=plotly.utils.PlotlyJSONEncoder)
+
+        return render(request, "HOMEPAGE.html", {'all_metrics':all_metrics, "current_metrics":current_metrics, "total_warnings":total_warnings, "graphJSON":graphJSON,"masterlist_count":masterlist_count,"climate_collection_count":climate_collection_count})
```
Comment (xhagrg) [inline dashboard/views.py]: Use multi line for longer lines.

File: dashboard/templates/base.html [hunk 6]
```diff
@@ -10,14 +10,15 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
 </head>
 
 <body>
 
 
 
     <!--Creating Responsive Menu Bar for Block Content-->
-    <navbar class="navbar">
+    <navbar class="navbar" style="z-index: 1">
```
Comment (xhagrg) [inline dashboard/templates/base.html:21]: Use css file for styling.

File: dashboard/static/css/style.css (hunk 1 above)
Comment (andrewdweis) [reply dashboard/static/css/style.css]: Resolved

**Pull request merged**